Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
 import './Card.css';
```

Comment:
quick note you dont need to import this css file because your importing it in a higher order component.